### ISSUE TITLE
Galaxy BamCompare region has colons ':' converted to 'X'

### ISSUE THREAD

**Issue by lparsons** [fidelram/deepTools#417]
When specifying a region using (e.g. `chr1:1:100`), I get an error from Galaxy because chromosome `chr1X1X100` doesn't exist.

**Comment by dpryan79** [comment]
@bgruening: Was there a magic fix for Galaxy character replacement stuff?

**Comment by dpryan79** [comment]
@lparsons: I think I found the problem. When we fixed #335 I think we inadvertently caused this bug. I've modified all of the wrappers on our public Galaxy server so I hope this will work again, but i'll need to restart Galaxy for this to take effect. There's a job running on it now that should be either finished or killed within the next ~1 hour, so I'll restart things after that. Hopefully that'll solve the issue.
